data/translations/zh/state-data.json

```diff
@@ -3,311 +3,411 @@
-    "ip_deadline": "選舉日前 29 天",
-    "online_deadline": "選舉日前 29 天",
-    "mail_postmarked_deadline": "選舉當日前 29 天郵戳為憑",
-    "confirm_registration_link": "https://indianavoters.in.gov/"
+    "ip_deadline": "2022年10月11日星期二",
+    "online_deadline": "2022年10月11日星期二",
+    "mail_postmarked_deadline": "郵戳為憑（2022年10月11日星期二）",
+    "confirm_registration_link": "https://indianavoters.in.gov/",
+    "default_ip_deadline": "選舉日前 29 天",
+    "default_online_deadline": "選舉日前 29 天",
+    "default_mail_postmarked_deadline": "選舉當日前 29 天郵戳為憑"
   },
   "ia": {
     "hp_link": "https://sos.iowa.gov/",
     "registration_link": "https://mymvd.iowadot.gov/Account/Login?ReturnUrl=%2fVoterRegistration",
     "more_info_link": "https://sos.iowa.gov/elections/voterinformation/voterregistration.html",
-    "ip_deadline": "選舉日之前（包括選舉日）都可以使用",
-    "online_deadline": "選舉日前 15 天",
-    "mail_postmarked_deadline": "選舉當日前 15天郵戳為憑",
-    "confirm_registration_link": "https://sos.iowa.gov/elections/voterreg/regtovote/search.aspx"
+    "ip_deadline": "2022年11月8日星期二",
+    "online_deadline": "2022年10月24日星期六",
+    "mail_postmarked_deadline": "郵戳為憑（2022年10月24日星期六）",
+    "confirm_registration_link": "https://sos.iowa.gov/elections/voterreg/regtovote/search.aspx",
+    "default_ip_deadline": "選舉日之前（包括選舉日）都可以使用",
+    "default_online_deadline": "選舉日前 15 天",
+    "default_mail_postmarked_deadline": "選舉當日前 15天郵戳為憑"
   },
   "ks": {
     "hp_link": "https://sos.ks.gov/elections/elections.html",
     "registration_link": "https://www.kdor.ks.gov/Apps/VoterReg/Default.aspx",
     "more_info_link": "https://sos.ks.gov/elections/voter-information.html",
-    "ip_deadline": "選舉日前 21 天",
-    "online_deadline": "選舉日前 21 天",
-    "mail_postmarked_deadline": "選舉當日前 21天郵戳為憑",
-    "confirm_registration_link": "https://myvoteinfo.voteks.org/voterview/"
+    "ip_deadline": "2022年10月18日星期二",
+    "online_deadline": "2022年10月18日星期二",
+    "mail_postmarked_deadline": "郵戳為憑（2022年10月18日星期二）",
+    "confirm_registration_link": "https://myvoteinfo.voteks.org/voterview/",
+    "default_ip_deadline": "選舉日前 21 天",
+    "default_online_deadline": "選舉日前 21 天",
+    "default_mail_postmarked_deadline": "選舉當日前 21天郵戳為憑"
   },
   "ky": {
     "hp_link": "https://elect.ky.gov/Pages/default.aspx",
     "registration_link": "https://vrsws.sos.ky.gov/ovrweb/",
     "more_info_link": "https://elect.ky.gov/Resources/Pages/Registration.aspx",
-    "ip_deadline": "選舉日前的第四個星期一",
-    "online_deadline": "選舉日前的第四個星期一",
-    "mail_received_deadline": "必須在選舉日前的第四個星期一之前收到",
-    "confirm_registration_link": "https://vrsws.sos.ky.gov/VIC/"
+    "ip_deadline": "2022年10月11日星期二",
+    "online_deadline": "2022年10月11日星期二",
+    "mail_received_deadline": "郵戳為憑（2022年10月11日星期二）",
+    "confirm_registration_link": "https://vrsws.sos.ky.gov/VIC/",
+    "default_ip_deadline": "選舉日前的第四個星期一",
+    "default_online_deadline": "選舉日前的第四個星期一",
+    "default_mail_postmarked_deadline": "必須在選舉日之前的第四個星期一之前蓋上郵戳"
```

Comment:
Update to `default_mail_received_deadline` We can't have both a postmarked and received value.

Reply:
Fixed